bin/improver-standardise

```diff
@@ -0,0 +1,179 @@
+
+    parser.add_argument("--fix_float64", action='store_true', default=False,
+                        help="Check and fix cube for float64 data. Without "
+                             "this option a warning will be raised if float64 "
+                             "data is found but no fix applied.")
+
+    parser.add_argument("--check_float64", action='store_true', default=False,
+                        help="Check the cube for float64 data. If float64 "
+                             "data is found a warning will be raised but "
+                             "no fix applied.")
+
+    parser.add_argument("--regrid", action='store_true', default=False,
+                        help="regrid cube.....")
+
+    parser.add_argument("--nearest", action='store_true', default=False,
+                        help='If True, regridding will be performed using '
+                             'iris.analysis.Nearest() instead of Linear().'
+                             'Use for less continuous fields, '
+                             'e.g. precipitation.')
+
+    parser.add_argument("--extrapolation_mode", default='nanmask',
+                        help='Mode to use for extrapolating data into regions '
+                             'beyond the limits of the source_data domain. '
+                             'Modes are:'
+                             'extrapolate - The extrapolation points will '
+                             'take their value from the nearest source point. '
+                             'nan - The extrapolation points will be be '
+                             'set to NaN. '
+                             'error - A ValueError exception will be raised, '
+                             'notifying an attempt to extrapolate. '
+                             'mask  - The extrapolation points will always be '
+                             'masked, even if the source data is not a '
+                             'MaskedArray. '
+                             'nanmask - If the source data is a MaskedArray '
+                             'the extrapolation points will be masked. '
+                             'Otherwise they will be set to NaN. '
+                             'Defaults to nanmask.')
+
+    parser.add_argument("--json_file", metavar="JSON_FILE", default=None,
+                        help='Filename for the json file containing required '
+                             'changes to the metadata. Defaults to None.')
+
+    args = parser.parse_args()
+
+    # source file data path is a mandatory argument
+    source_data = load_cube(args.source_data_filepath)
+
+    # Re-grid with options:
+    # if a target grid file, output file path and regrid options have been
+    # specified, then regrid optionally applying float64 data check, metadata
+    # change, iris nearest and extrapolation mode as required
+    if args.target_grid_filepath and args.output_filepath and args.regrid:
```

Comment:
It might be easier if the presence of the `args.output_filepath` is tested prior to this if statement. That would reduce this if statement to being:
```python
if args.target_grid_filepath:
...
elif args.json_file:

elif args.fix_float64:
...
```
The presence of the output_filepath could be checked using something like:
```python
if args.target_grid_filepath or args.json_file or args.fix_float64:
    if not args.output_filepath:
       msg = ""
       raise ValueError(msg)
```
Unfortunately at the moment, it's possible to specify just args.target_grid_filepath without an args.output_filepath, and the CLI won't do anything.

Reply:
I see what you mean and will make these changes.